docs/articles/peer-monitoring-sets.md

```diff
@@ -0,0 +1,152 @@
+monitoring sets in the other's TROLIE in order to monitor ratings, as described in the operations 
+[getLimitsForecastSnapshot](../spec#tag/Forecasting/operation/getLimitsForecastSnapshot) and
+[getRealTimeLimits](../spec#tag/Real-Time/operation/getRealTimeLimits).
+
+![Peers](../images/Seam.excalidraw.png)
+
+As indicated by the diagram, the actual seam between the RCs can be derived where the two monitoring sets
+overlap.  For these resources, both RCs are producing a rating.  
+
+# Monitoring Set Reconciliation Process
+
+Consider the perspective of RC A given the above diagram.  RC A must be aware of both its own monitoring set
+in RC B's TROLIE, which will be referred to as monitoring set `B-in` in the following examples.   In addition,
+The RC A TROLIE can gain insight by specifically being aware of the monitoring set in its own database that 
+RC B is using to monitor RC A's footprint.  This will be referred to as `B-out` in the following examples.  
+
+Visibility into the monitoring set itself in either system is exposed via the 
+[getMonitoringSet](../spec#tag/Monitoring-Sets/operation/getMonitoringSet) operation.  Given this capability, 
+TROLIE for RC A may do the following:
+
+* Automatically determine the overlapping aliases between models. 
+* Automatically derive the actively configured seam between RCs, to use for debugging and monitoring.
+
+This automation should be triggered by an update to either the `B-in` or `B-out` monitoring 
+sets.  The process to update `B-out` is an implementation internal.  Updates to `B-in` are 
+facilitated by an independent polling mechanism in the RC A TROLIE, using the 
+[conditional GET](./conditional-GET.md) pattern against `getMonitoringSet`.  When 
+`getMonitoringSet` returns a 200 status (as opposed to 304), indicating a new version, then this triggers 
+reconciliation.  This is illustrated in the data flow below.  Note that "Monitoring Set"
+is abbreviated to "MS" for readability.    
+
+```mermaid
+
+flowchart
+   
+  rcb[RC B TROLIE]  
+  msp[[MS Poller]]
+  bin[(B-in MS)]
+  bout[(B-out MS)]
+  msr[[MS Reconciler]]
+  mappedIds[(RC B Mapped Ids)]
+  seam[(Seam MS)]
+  
+  msp -- Poll for new version of B-In --> rcb
+  rcb -- Return new version --> msp
+  msp ---> bin
+
+  bin -- new version --> msr
+  bout -- new version --> msr
+
+  msr -- update --> mappedIds
+  msr -- update --> seam
+
+```
+
+The above diagram illustrates two outputs of a reconciliation process, both of which should be
+stored in the TROLIE server's database:
+
+* Mappings of remote identifiers (the identifiers expected by RC B in this case)
+* The seam monitoring set
+
+# Automatic Resource Alias Mapping
+In practice, most Transmission Providers have at least the following identifiers for each resource:
+
+* An identity for the resource in a modeling system and/or asset registration system.  Since many modeling tools are CIM-based, this is typically a master resource identifier (MrID) in the form of a UUID.  
```

Comment:
```suggestion
* An identity for the resource in a modeling system and/or asset registration system.  Since many modeling tools are CIM-based, this is typically a master resource identifier (mRID) in the form of a UUID.  
```